src/api/cristinApi.ts

```diff
@@ -149,3 +149,17 @@ export const fetchProject = async (projectId: string) => {
   });
   return fetchProjectRespone.data;
 };
+
+export const uploadProfilePicture = async (userId: string, base64String: string) =>
```

Comment:
`user` bruker vi mest om vår interne bruker, mens vi bruker `person` om de som er i cristin, så foreslår renaming her. I metoden undet brukes `cristinId` også, så det er enda et argument 
```suggestion
export const uploadProfilePicture = async (cristinId: string, base64String: string) =>
```